cloud/aws/templates/aws_oidc/monitoring.tf

```diff
@@ -1,11 +1,10 @@
-data "aws_caller_identity" "current" {}
-
 resource "aws_prometheus_workspace" "metrics" {
-  alias = "CiviForm_metrics"
+  alias = "${var.app_prefix}-CiviForm_metrics"
 }
 
 resource "aws_grafana_workspace" "CiviForm_metrics" {
-  name                     = "CiviFormMetrics"
+  name                     = "${var.app_prefix}-CiviFormMetrics"
```

Comment:
For consistency maybe {var.app_previx}-civiform-metrics unless there is a reason why we need to use different casing here.

Reply:
done